plugin-testing-framework/definitions/replace.d.ts

```diff
@@ -0,0 +1,13 @@
+interface ReplaceOptions {
+	regex: string;
+	replacement: string;
+	paths: Array<string>;
+	recursive: boolean;
+	silent: boolean;
```

Comment:
i suppose but afaik this was retrieved from definitelytyped or something, so correct me if i'm wrong but there's no point in changing it because we'd have to do the same updates when a new release of replace comes out or try to merge our changes into their typings file